### ISSUE TITLE
Error getting installed games for provider: The system cannot find the file specified. (os error 2)

### ISSUE THREAD

**Issue by drhouse** [Raicuparta/rai-pal#126]
Here are the contents of my log file after running version 0.3.7, I tried 'Fix list' and 'Reset settings to default', but I keep getting:

> Error getting installed games for provider: The system cannot find the file specified. (os error 2)

```
[2024-01-03][14:06:10][INFO][rai_pal::analytics] Sending AnalyticsPayload { client_id: "0f1967bc-d635-4440-ba08-67250cd54957", timestamp_micros: 1704290770837210, non_personalized_ads: true, events: [AnalyticsEvent { name: StartApp, params: AnalyticsEventParams { data: "", app_version: "0.3.7" } }] }
[2024-01-03][14:06:10][INFO][rai_pal::debug]          0ms # get mod loader map
[2024-01-03][14:06:10][INFO][rai_pal::debug]          4ms # refresh local mods
[2024-01-03][14:06:10][INFO][rai_pal::analytics] Successfully Sent Analytics Event StartApp for 
[2024-01-03][14:06:10][INFO][rai_pal::debug]         57ms # set up provider (Steam)
[2024-01-03][14:06:10][INFO][rai_pal::debug]          0ms # set up provider (Epic)
[2024-01-03][14:06:10][INFO][rai_pal::debug]          0ms # set up provider (Gog)
[2024-01-03][14:06:10][INFO][rai_pal::debug]          0ms # set up provider (Xbox)
[2024-01-03][14:06:10][INFO][rai_pal::debug]          0ms # set up provider (Manual)
[2024-01-03][14:06:10][INFO][rai_pal::debug]         57ms # get provider map
[2024-01-03][14:06:10][INFO][rai_pal::debug]          0ms # get Xbox installed games (0 total)
[2024-01-03][14:06:10][ERROR][rai_pal::events] Error: Error getting installed games for provider: The system cannot find the file specified. (os error 2)
[2024-01-03][14:06:11][INFO][rai_pal::debug]        564ms # get Steam installed games (31 total)
[2024-01-03][14:06:11][INFO][rai_pal::debug]         20ms # get Epic installed games (0 total)
[2024-01-03][14:06:11][INFO][rai_pal::debug]          0ms # get Gog installed games (0 total)
[2024-01-03][14:06:11][ERROR][rai_pal::game_engines::unity] Failed to get Unity version: Failed to retrieve Unity version from asset `\\?\N:\Games\Endlight2\Endlight\Endlight\Endlight_Data\globalgamemanagers`
[2024-01-03][14:06:12][INFO][rai_pal::debug]        880ms # get Manual installed games (24 total)
[2024-01-03][14:06:12][INFO][rai_pal::debug]         12ms # get installed game map + update game mods
[2024-01-03][14:06:12][INFO][rai_pal::debug]         57ms # refresh remote mods
[2024-01-03][14:06:12][INFO][rai_pal::debug]         39ms # update game mods
[2024-01-03][14:06:12][INFO][rai_pal::debug]        185ms # get owned games (79 total)
```

**Comment by Raicuparta** [comment]
The error should be harmless, it will be gone on the next update. Are you seeing any actual problems in the app besides the error message popping up?

**Comment by Raicuparta** [comment]
Should be fixed in 0.4.0
